TreeMaker/python/doZinvBkg.py

```diff
@@ -63,7 +63,6 @@ def reclusterZinv(process, geninfo, residual, cleanedCandidates, suff, fastsim,
             "NumChadrons"          ,
         )
     )
-    
     process.JetsPropertiesAK8Clean.prunedMass = cms.vstring('ak8PFJetsCHSCleanPrunedMass')

```

Comment:
make the corresponding AK8 changes here (wrt my comments for the main collection, e.g. CHS -> Puppi, etc.)